src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { Component, Fragment } from 'react';
+import { Grid, Form, Input, Field, Button, Message } from '@b-design/ui';
+import DrawerWithFooter from '../../../../components/Drawer';
+import { updateUser } from '../../../../api/users';
+import type { LoginUserInfo } from '../../../../interface/user';
+import { checkName, checkUserPassword, checkUserEmail } from '../../../../utils/common';
+import Translation from '../../../../components/Translation';
+import i18n from '../../../../i18n';
+
+type Props = {
+  userInfo?: LoginUserInfo;
+  onClose: () => void;
+};
+
+type State = {
+  isLoading: boolean;
+};
+
+class EditPlatFormUserDialog extends Component<Props, State> {
+  field: Field;
+  constructor(props: Props) {
+    super(props);
+    this.field = new Field(this);
+    this.state = {
+      isLoading: false,
+    };
+  }
+
+  componentDidMount() {
+    const { userInfo = { name: '', alias: '', email: '' } } = this.props;
+    const { name, alias, email } = userInfo;
+    this.field.setValues({
+      name,
+      alias,
+      email,
+    });
+  }
+
+  onUpdateUser = async () => {
+    this.field.validate((error: any, values: any) => {
+      if (error) {
+        return;
+      }
+      const { name, alias, email, password } = values;
+      const params = {
+        name,
+        alias,
+        email,
+        password,
+      };
+      this.setState({
+        isLoading: true,
+      });
+      updateUser(params)
+        .then((res) => {
+          if (res) {
+            Message.success(<Translation>Update user success</Translation>);
+            this.props.onClose();
+          }
+        })
+        .finally(() => {
+          this.setState({
+            isLoading: false,
+          });
+        });
+    });
+  };
+
+  showTitle() {
+    return i18n.t('Edit User');
```

Comment:
Reset the password and email for the administrator account.

Reply:
done